File: Exercicios_Em_Python_CV/desafio_006.py
#Crie um algoritmo que leia um número e mostre o seu dobro, triplo e raiz quadrada
num = int(input('Digite um número inteiro:'))
print('Vamos obter o dobro do número {}:{}'.format(num, num*2))
print('Vamos obter o triplo do número {}:{}'.format(num,num*3))
print('Vamos obter a raiz quadrada de {}:{}'.format(num,num**(1/2)))
print('Vamos obter a raiz quadrada de {}:{}'.format(num,pow(num,(1/2))))

File: Exercicios_Em_Java_CV/Desafio006.java
package fundamentos;

import java.util.Scanner;

/*
 * Crie um algoritmo que leia um número e mostre:dobro,triplo e raiz
 */
public class Desafio006 {
	public static void main(String[] args) {
		Scanner scanner = new Scanner(System.in);
		int num1 = scanner.nextInt();
		System.out.printf("Dobro:%s, triplo:%s, raiz:%s",num1*2,num1*3,Math.sqrt(num1));
		System.out.printf("Dobro:%s, triplo:%s, raiz:%s",num1*2,num1*3,Math.pow(num1,1.0/2.0));

		scanner.close();
	}

}
